app/assets/stylesheets/application/styles/_event_new.scss

```diff
@@ -0,0 +1,113 @@
+.event-new {
+  @include container;
+  margin: 24px auto 0;
+  padding: 0 16px 48px;
+
+  @include breakpoint($tablet) {
+    margin-top: 48px;
+    padding-bottom: 64px;
+  }
+}
+
+.event-new__date {
+  display: inline-block;
+  box-sizing: border-box;
+  padding-right: 8px;
+  width: 50%;
+
+  .event-new__text-input {
+    height: 35px;
+  }
+}
+
+.event-new__time {
+  display: inline-block;
+  box-sizing: border-box;
+  padding-left: 8px;
+  width: 50%;
+
+  .event-new__text-input {
+    height: 35px;
+  }
+}
+
+.event-new__price {
+  display: inline-block;
+  box-sizing: border-box;
+  padding-right: 8px;
+  width: 50%;
+}
+
+.event-new__seats {
+  display: inline-block;
+  box-sizing: border-box;
+  padding-left: 8px;
+  width: 50%;
+}
+
+.event-new__text-input {
+  @extend %form-input;
+  display: block;
+}
+
+.event-new__text-label {
+  @extend %primary-font;
+  box-sizing: border-box;
+  display: block;
+  margin-bottom: 2px;
+  margin-top: 8px;
+}
+
+.event-new__form-divider {
+  border-top: 1px solid $light-gray-color;
+  padding-top: 16px;
+  margin-bottom: 8px;
+  margin-top: 16px;
+}
+
+.event-new__description-input {
+  @extend %form-input;
+  display: block;
+  height: 6em;
+  resize: vertical;
+}
+
+.event-new__menu-input {
+  @extend %form-input;
+  display: block;
+  height: 10em;
+  resize: vertical;
+}
+
+.event-new__photo-input {
+  @extend %form-input;
+  margin-top: 8px;
+}
+
+.event-new__primary-photo {
+  display: block;
+  height: 100%;
+  object-fit: cover;
+  margin-top: 8px;
+  max-width: 1024px;
+  max-height: 678px;
+  width: 100%;
+}
+
+.event-new__photos {
+  margin-top: 8px;
+}
+
+.event-new__photos img {
+  height: 100%;
+  object-fit: cover;
```

Comment:
Properties should be ordered height, margin-top, max-height, max-width, object-fit, width